### ISSUE TITLE
[Bug] OAuth - Facebook Provider (400 Bad Request)

### ISSUE THREAD

**Issue by chrisburton** [statamic/v2-hub#1467]
I'm currently using Twitter, Github, Instagram as providers for OAuth. All the providers work as intended except when I try to add Facebook as an additional provider. I receive the following error:

```
Client error: `GET https://graph.facebook.com/v2.8/me?access_token=&fields=name,email,gender,verified,link&appsecret_proof=SOME_RANDOM_NUMBERS` resulted in a `400 Bad Request` response:
{"error":{"message":"An active access token must be used to query information about the current user.","type":"OAuthExce (truncated...)
```

I did a quick google search which stated that the Facebook API updated therefore, a composer update is required to fix this.

Source: https://stackoverflow.com/questions/38451505/an-active-access-token-must-be-used-to-query-information-about-the-current-user

**Comment by chrisburton** [comment]
Also, see: https://github.com/laravel/socialite/commit/441e0d467319044e4913b1748382d427795416e9

**Comment by chrisburton** [comment]
Just confirmed the above update works if I manually edit the file.

**Comment by jasonvarga** [comment]
Fixed in 2.5.11

**Comment by tolawho** [comment]
@jasonvarga I can't install facebook provider  "composer require socialiteproviders/facebook". This package no longer compatible with statamic v2
